kinetic/distribution.yaml

```diff
@@ -10435,6 +10435,22 @@ repositories:
       url: https://github.com/team-vigir/vigir_step_control.git
       version: master
     status: maintained
+  vision_msgs:
+    doc:
+      type: git
+      url: https://github.com/Kukanani/vision_msgs.git
+      version: kinetic-devel
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/Kukanani/vision_msgs-release.git
+      version: 0.0.1-0
+    source:
+      test_pull_requests: true
```

Comment:
Just checking: Did you follow the instructions from http://wiki.ros.org/buildfarm/Pull%20request%20testing to grant access to ros-pull-request builder to your source repository ?

Reply:
I just accepted the invitation.